elastic/kibana
# Add possibility to update threat_indicator_path for prebuilt rule

## Add the possibility to update threat_indicator_path for the prebuilt rule.


### How to reproduce the bug:

We will update this file -`x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/threat_intel_module_match.json`

- checkout master branch
- install all prebuilt rules
- open the _Threat Intel Filebeat Module Indicator Match_
- check that in the network panel the request for this rule  has  `threat_indicator_path`equals ""
- change `threat_indicator_path` to any value
- increase the version
- save file
- you should see this button
<img width="834" alt="Screenshot 2021-10-28 at 19 54 07" src="https://user-images.githubusercontent.com/7609147/139309429-80118533-4ad8-40ef-9fb4-1f1792e95dd6.png">

- update rules
- check Indicator math rule again in the network request
- `threat_indicator_path`should  be the empty string -  ""

### Check out branch

- Go to `x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/threat_intel_module_match.json`
- change `threat_indicator_path` to any value
- increase the version
- you should see this button
<img width="834" alt="Screenshot 2021-10-28 at 19 54 07" src="https://user-images.githubusercontent.com/7609147/139309429-80118533-4ad8-40ef-9fb4-1f1792e95dd6.png">

- update rules
- check _Threat Intel Filebeat Module Indicator Match_ rule again in the network request
- `threat_indicator_path`should  has a value which you enter before





### For maintainers
